features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | identificatiecode nummeraanduiding (11.90) | <nummeraanduidingIdentificatie>    |
+      | locatiebeschrijving (12.10)                | <locatiebeschrijving>              |
+      | land.code                                  | <land>                             |
+      | land.omschrijving                          | <land omschrijving>                |
+      | datum aanvang adres buitenland (13.20)     | <datumAanvangAdresBuitenland>      |
+      | regel 1 adres buitenland (13.30)           | <regel1>                           |
+      | regel 2 adres buitenland (13.40)           | <regel2>                           |
+      | regel 3 adres buitenland (13.50)           | <regel3>                           |
+      | ingangsdatum geldigheid (85.10)            | datumIngangGeldigheid              |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | <burgerservicenummer>           |
+      | fields              | verblijfplaats.datumVan         |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                              | waarde                             |
+      | verblijfadres.straat              | <straat>                           |
+      | verblijfadres.naamOpenbareRuimte  | <straat>                           |
+      | verblijfadres.huisnummer          | <huisnummer>                       |
+      | verblijfadres.postcode            | <postcode>                         |
+      | verblijfadres.woonplaats          | <woonplaats>                       |
+      | verblijfadres.locatiebeschrijving | <locatiebeschrijving>              |
+      | verblijfadres.land.code           | <land>                             |
+      | verblijfadres.land.omschrijving   | <omschrijving>                     |
+      | verblijfadres.regel1              | <regel1>                           |
+      | verblijfadres.regel2              | <regel2>                           |
+      | verblijfadres.regel3              | <regel3>                           |
+      | adresseerbaarObjectIdentificatie  | <adresseerbaarObjectIdentificatie> |
+      | nummeraanduidingIdentificatie     | <nummeraanduidingIdentificatie>    |
+      | datumAanvangAdreshouding          | <datumAanvangAdreshouding>         |
+      | datumAanvangAdresBuitenland       | <datumAanvangAdresBuitenland>      |
+
+      Voorbeelden:
+      | burgerservicenummer | functieAdres | functieAdres omschrijving | datumAanvangAdreshouding | straat    | huisnummer | postcode | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie | locatiebeschrijving        | land | land omschrijving | datumAanvangAdresBuitenland | regel1            | regel2       | regel 3 |
```

Comment:
In de 2e rij heeft `datumAanvangAdreshouding` geen waarde, wellicht bewust om een datumOnbekend af te dwingen. Echter in zowel rij 3 als 4 heeft datumAanvangAdresBuitenland geen waarde. Lijkt me dat deze in minimaal een van die rijen een waarde moet hebben.

Reply:
dat hoort wel. toegevoegd